tests/rbd_system.py

```diff
@@ -15,18 +13,19 @@ def run(**kw):
     client_node = rgw_client_nodes[0]
     # cleanup any existing stale test dir
     test_folder = 'rbd-tests'
-    client_node.exec_command(cmd='rm -rf ' + test_folder)
+    client_node.exec_command(cmd='sudo rm -rf ' + test_folder)
```

Comment:
install python3 and create virtualenv based on python3. activate the env and then install the packages/